app/controllers/posts_controller.rb

```diff
@@ -1,12 +1,31 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id])
-    @posts = @user.posts.includes(:comments)
+    @user = current_user
+    @posts = @user.posts
   end
 
   def show
-    @user = User.find(params[:user_id])
-    @post = @user.posts.find(params[:id])
-    @like = Like.where(user_id: current_user.id, post_id: params[:id]).exists?
+    @post = Post.find(params[:id])
+  end
+
+  def new
+    @post = Post.new
+    # render :new, locals: { post: post }
+  end
+
+  def create
+    @user = current_user
+    # @post = current_user.posts.create(title: params[:tile], text: params[:text])
+    @post = Post.new(params.require(:post).permit(:title, :text))
+    # post.users = user
+    # post.users_id = user.id
```

Comment:
- [ ] I will suggest you remove unused codes from your codebase. The reasons are given below.